tests/test_policy.py

```diff
@@ -777,6 +777,12 @@ def test_policy_with_role_complete(self):
 
     def test_policy_variable_interpolation(self):
 
+        import os
+
+        os.environ['C7N_VAR_email'] = 'me@example.com'
```

Comment:
to avoid pollution of other unit tests, use the method self.change_environment()